smslibrary/src/main/java/com/eis/smslibrary/listeners/SMSReceivedServiceListener.java

```diff
@@ -0,0 +1,47 @@
+package com.eis.smslibrary.listeners;
+
+import android.app.IntentService;
+import android.content.Intent;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSReceivedBroadcastReceiver;
+
+/**
+ * Abstract class to implement in order to wake up the service when a message is received
+ *
+ * @author Luca Crema, Alberto Ursino, Marco Tommasini, Alessandra Tonin
+ * @since 29/11/2019
+ */
+public abstract class SMSReceivedServiceListener extends IntentService {
```

Comment:
Is this still needed after removing the service?

Reply:
Sure it is, the app is dead and no activity can be woken up, so the app has to use a service that will run in the background without the need of the whole app to be loaded

Reply:
Ok, thanks